packages/react/src/products/hooks/__tests__/useProductDetails.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { fetchProductDetails } from '@farfetch/blackout-redux/products';
+import {
+  mockGroupedEntries,
+  mockProductId,
+  mockProductsState,
+} from 'tests/__fixtures__/products';
+import { mockStore } from '../../../../tests/helpers';
+import { Provider } from 'react-redux';
+import { renderHook } from '@testing-library/react-hooks';
+import React from 'react';
+import useProductDetails from '../useProductDetails';
+
+jest.mock('@farfetch/blackout-redux/products', () => ({
+  ...jest.requireActual('@farfetch/blackout-redux/products'),
+  fetchProductDetails: jest.fn(() => ({ type: 'get' })),
+}));
+
+// eslint-disable-next-line react/prop-types
+const ReduxProviderWrapper =
+  storeState =>
+  ({ children }) => {
+    const finalStoreState = {
+      ...mockProductsState,
+      ...storeState,
+    };
+
+    return <Provider store={mockStore(finalStoreState)}>{children}</Provider>;
+  };
+
+const renderHookWithRedux = (props, storeState = undefined) => {
```

Comment:
What about having a `mockProductDetailsState` as the default of the `storeState`? Not only it's consistent with the other hooks "strategy", but it also gets simpler and shorter (you stop needing `ReduxProviderWrapper` and an assembled `finalStoreState`).

Reply:
And for the sake of consistency, I'd call this `getRenderedHook` instead of `renderHookWithRedux`.